### ISSUE TITLE
Signature verification failed when posting idea with Android SDK

### ISSUE THREAD

**Issue by kbroulik** [uservoice/uservoice-android-sdk#220]
When I try to post a new idea in UserVoice using the Android SDK I get the following error:
```
E/com.uservoice.uservoicesdk(25611): 401 -- {"errors":{"type":"unauthorized","message":"Unterschrieber Client erforderlich: Signature verification failed"}}
```
When using the same site, key and secret on iOS, it works just fine. I tried doing it with no key or with a newly generated key, but neither work. It also made no difference whether I provided a user email or not.
```
m_config = new Config(site, key, secret);
m_config.identifyUser(guid, username, email);
if (forumId != 0) {
    m_config.setForumId(forumId);
}
if (topicId != 0) {
    m_config.setTopicId(topicId);
}
UserVoice.init(m_config, m_activity);
UserVoice.launchUserVoice(m_activity);
```

**Comment by bratislavdamnjanovic** [comment]
Same problem here

**Comment by austintaylor** [comment]
Fixed in master + 1.2.5.
